src/modules/account/ui/ActionField/ActionFieldLayout.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, ReactNode } from 'react';
+
+type ActionFieldLayoutProps = {
+  label: string | ReactNode;
+  actionButton: ReactNode;
+  showActionButton: boolean;
+  className?: string;
+};
+
+const ActionFieldLayout: FC<ActionFieldLayoutProps> = ({
+  children,
+  className,
+  label,
+  actionButton,
+  showActionButton,
+}) => (
+  <article tw="pt-0 pb-2xl" className={className}>
+    <header tw="flex justify-between items-baseline">
+      <h3 tw="font-semibold">{label}</h3>
+
+      {showActionButton && <>{actionButton}</>}
```

Comment:
If `showActionButton && actionButton` is confusing, change to `renderedActionButton`.